### ISSUE TITLE
Links must have discribe text

### ISSUE THREAD

**Issue by JayneSoraya** [JayneSoraya/meu-portfolio#9]
Ensure links have discernible text

To solve this problem, you need to fix at least (1) of the following:
Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
And fix the following:
Element is in tab order and does not have accessible text

Element Location:
.bg-neutral-900\/50.rounded-2xl.shadow-xl:nth-child(5) > .aspect-video.overflow-hidden.relative > .justify-center.items-center.duration-500 > .translate-y-4.group-hover\:translate-y-0.hover\:bg-purple-500

<a href="https://github.com/JayneSoraya/Empreenda-Senac/blob/main/README.md" target="_blank" rel="noopener noreferrer" class="bg-purple-600 text-white p-4 rounded-full shadow-lg transform translate-y-4 group-hover:translate-y-0 transition-transform hover:bg-purple-500 z-20">
